src/ingest_validation_tools/directory-schemas/codex-v0.yaml

```diff
@@ -12,7 +12,7 @@ files:
     is_qa_qc: True
     required: False
   - 
-    pattern: '(raw|processed)/config\.txt|(src_[^/]*|drv_[^/]*|extras)/[sS]egmentation\.json' 
+    pattern: '(raw|processed)/config\.txt|(src_[^/]*|drv_[^/]*)/[sS]egmentation\.json' 
```

Comment:
We still have some codex data uploads that uses the raw folder for this, not sure if that is the case anymore or if it should be allowed.

Reply:
@sunset666 this is what @pdblood asked me to do. @bhonick can you confirm this is correct? cc @mruffalo